test/unit_Polytope.jl

```diff
@@ -350,6 +350,12 @@ if test_suite_polyhedra
         @test BallInf(N[0, 0], N(1)) ⊆ P
         @test !(BallInf(N[0, 0], N(1.01)) ⊆ P)
 
+        # concrete minkowski sum
+        B = convert(VPolytope, BallInf(N[0, 0, 0], N(1)))
+        X = minkowski_sum(B, B, backend=CDDLib.Library())
```

Comment:
It doesn't work with the default library unfortunately (here the command runs forever). If you have another idea to substitute this one let's change it, although the BallInf is already quite "simple".